daffodil-lib/src/main/scala/org/apache/daffodil/xml/PrettyPrinter.scala

```diff
@@ -15,9 +15,8 @@
  * limitations under the License.
  */
 
-package org.apache.daffodil.xml.scalaLib
+package org.apache.daffodil.xml
```

Comment:
Everything beyond this line seems unrelated to NOTICE/LICENSE.

Reply:
Part of the below changes removes the Utility.scala file, which came from the Scala library. We don't really use it anymore so I figured it was better to remove it so we do not need to include it in the LICENSE file that mentioned the mentions the other scala lib file (UniquenessCache.scala).

The PrettyPrinter scala file (which is the remaining of the below changes) originally came from the scala library, but we have since completely rewritten it (basically only the function names still exist), so I'm not sure it is considered a derivative work anymore. The rest of the below changes were to just make it clear that it PrettyPretty is not a scala lib file by moving it out of the scalaLib directory. This required changing some imports. So the below changes are more about removing the potential for someone to think PrettyPretty is a scala lib file with an incorrect license. That's somewhat related to this patchset, so thought it reasonable to include it here.

Reply:
Ok, sounds fair enough.